Package.swift

```diff
@@ -4,17 +4,22 @@
 import PackageDescription
 
 let package = Package(
-	name: "ATTNSDKFramework",
-	platforms: [.iOS(.v12)],
-	products: [
-		.library(name: "ATTNSDKFramework", targets: ["ATTNSDKFramework"])
-	],
-	targets: [
-		.target(
+  name: "ATTNSDKFramework",
+  platforms: [.iOS(.v12)],
+  products: [
+    .library(name: "ATTNSDKFramework", targets: ["ATTNSDKFramework", "ATTNSDKFrameworkObjc"])
+  ],
+  targets: [
+    .target(
       name: "ATTNSDKFramework",
       path: "Sources",
-      resources: [ .process("Resources") ],
+      resources: [ .process("Resources") ]
+    ),
+    .target(
```

Comment:
For SPM, the Sources folder is causing errors if the Swift and Objc files are mixed. 
The workaround is to create a new target in the Package.swift. If the client decides not to use the new class with constants, they have to import the following:
```
import ATTNSDKFrameworkObjc
```